notebooks/04-Creating_ASDF_Files.ipynb

```diff
@@ -0,0 +1,753 @@
+{
```

Comment:
<p>Should include boolean type as part of the type list</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/asdf-format/scipy2022tutorial/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>